### ISSUE TITLE
ARIA-label given wrong for footer links

### ISSUE THREAD

**Issue by navaneethsnair1** [OpenLiberty/openliberty.io#3927]
<img width="428" alt="Screenshot 2025-01-03 at 11 43 43 AM" src="https://github.com/user-attachments/assets/289530db-d4a4-4fc1-8c6f-4a6f4b78398d" />

Same aria-label is given for multiple footer links causing accessibility violation.
